pkg/controller/seed-controller-manager/default-application-controller/controller.go

```diff
@@ -194,6 +196,12 @@ func (r *Reconciler) reconcile(ctx context.Context, cluster *kubermaticv1.Cluste
 			continue
 		}
 
+		if managedByKKP := applicationDefinition.Labels[appskubermaticv1.ApplicationManagedByLabel]; managedByKKP == appskubermaticv1.ApplicationManagedByKKPValue {
+			if appType := applicationDefinition.Labels[appskubermaticv1.ApplicationTypeLabel]; appType != appskubermaticv1.ApplicationTypeCNIValue {
+				applications = append(applications, applicationDefinition)
+			}
+		}
```

Comment:
Can you please explain a bit what we are trying to do here? IMO this is not required.